pkg/net/net.go

```diff
@@ -66,3 +69,46 @@ func NthIPInNetwork(network *net.IPNet, n int) (net.IP, error) {
 
 	return nil, errors.New("network does not contain enough IPs")
 }
+
+// DNSNames returns a default set of machine names. It includes the hostname,
+// and FQDN. If the kernel domain name is not set, no names will be returned.
```

Comment:
I'm wondering if we should do this now that I'm looking at it. If a domain name is not found, I don't think that we should ignore hostname as this code is now. Thoughts?

Reply:
I went with sticking the hostname in there even if the domain name isn't set. I can change back if we decide we don't want to do this.